oktapam/data_source_ad_connections_test.go

```diff
@@ -39,21 +37,31 @@ const testAccDatasourceADConnectionsConfigFormat = `
 data "oktapam_gateways" "gateways" {
 }
 
-resource "oktapam_ad_connection" "test_acc_ad_connection_1" {
+resource "oktapam_ad_certificate_request" "self_signed_cert" {
+  display_name = "test_acc"
+  common_name  = "test_acc"
+  type         = "self_signed"
+  details {
+	ttl_days = 1
+  }
+}
+
+resource "oktapam_ad_connection" "test_acc_ad_connection" {
 	 name                     = "%[1]s"
 	 gateway_id               = data.oktapam_gateways.gateways.gateways[0].id
-	 domain                   = "testacc.example.com"
```

Comment:
Ah gotcha, so the fact this was the same domain every time meant that the tests could clash?